proto/build_event_stream.proto

```diff
@@ -575,6 +575,7 @@ message File {
   int64 length = 6;
 }
 
+// BUILDBUDDY-SPECIFIC EVENT
```

Comment:
There is a section further down in the file with `// BUILDBUDDY-SPECIFIC EVENTS` already if you want to move this proto there instead

Reply:
Oh that's a much better idea.